packages/autocomplete/src/autocomplete.vue

```diff
@@ -230,7 +247,8 @@
         }
       },
       select(item) {
-        this.$emit('input', item[this.valueKey]);
+        this.selected = true;
```

Comment:
Does `selected` mean that you're in the process of selecting an item (but not yet selected)? If so that might be a better form of name to avoid confusing this with a "currently selected value", which is a common data property.